apps/examples/mediarecorder/mediarecorder_main.cpp

```diff
@@ -182,15 +183,25 @@ class MediaRecorderTest : public MediaRecorderObserverInterface
 				break;
 				case VOLUME_UP: {
 					std::cout << "VOLUME_UP" << std::endl;
-					if (mr.setVolume(mr.getVolume() + 1) == RECORDER_ERROR) {
+					int mVol = mr.getVolume();
+					if (mVol < 0) {
+						cout << "getVolume failed" << endl;
+						break;
+					}
+					if (mr.setVolume(mVol + 1) == RECORDER_ERROR) {
```

Comment:
Audio_manager handles volume between 0 and 10
We will add new enum for volume, I think we don't have to fix this for now.

Reply:
OK. if the volume enum will be redefined.
Currently, it is problematic because it can have minus values such as -6, -1 by returning from ret = get_output_audio_volume ();